packages/ember-auto-import/ts/package.ts

```diff
@@ -159,9 +159,13 @@ export default class Package {
 
   isEmberAddonDependency(name: string): boolean {
     if (!this.isAddonCache.has(name)) {
-      let packageJSON = require(resolve.sync(`${name}/package.json`, {
-        basedir: this.root
-      }));
+      let packagePath = resolvePackagePath(`${name}/package.json`, this.root);
+
+      if (packagePath === null) {
+        throw new Error(`${ this.name } tried to import "${name}" but the package was not resolvable from ${this.root}`);
```

Comment:
This error is added mostly to satisfy typing constraints (since `resolvePackagePath` may return `null` if the package is not found). This does **not** introduce additional cases where this would happen though, `resolve.sync` just threw an error internally if it couldn't resolve the package.